code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -66,6 +66,23 @@
 	gold_core_spawnable = 0
 
 /mob/living/simple_animal/pet/cat/Life()
+	if(!stat && !buckled)
+		if(prob(1))
+			emote("me", 1, pick("stretches out for a belly rub.", "wags its tail."))
+			icon_state = "[icon_living]_rest"
+			resting = 1
+		else if (prob(1))

```

Comment:
Conditional probability means that these get less and less likely the further down you go (1%, 0.99%, 0.9801%, etc). Granted, not that big of a deal, but probably not intended. Should probably either do a pick/prob chain or `switch(pick(rand(1, 100)))`.